### ISSUE TITLE
Jailbird doesn't set you to arrest when you are a Stowaway

### ISSUE THREAD

**Issue by ModDownloading** [goonstation/goonstation#4293]
**Describe the bug**
A more-researched version of my previous bug report #4230, I did some testing with the traits and determined that I wasn't being set to arrest because I had the Stowaway trait in addition to Jailbird. Other effects like Conspiracy Theorist's random conspiracies might also not appear if you have a origin trait, but I don't know that last part for sure since it's chance based.

**To Reproduce**
1. Take Jailbird and Stowaway traits
2. Join the round
3. Run towards securitron
4. Notice you are not set to arrest

**Expected behavior**
I am set to arrest while starting with Jailbird and Stowaway

**Additional context**
The Pilot trait DOES work with Jailbird, which makes it seem like it's an issue with Stowaway and not origin traits in general.

**Comment by Carbadox** [comment]
I frequently play Stowaway and Jailbird and this hasn't been an issue for me whatsoever

**Comment by ModDownloading** [comment]
I downloaded the source code and compiled it and was able to get the problem to show up on there.

The method I used was:
1st: Create a character with just Jailbird, I am set to arrest roundstart.
2nd: Change the character to have the plethora of traits I use ingame (Jailbird, Survivalist, Addictive Personality, Reptilian, Stowaway and Conspiracy Theorist), I am not set to arrest roundstart.
3rd: Remove all the traits except for Jailbird and Stowaway, I am not set to arrest roundstart.

This is how I was able to determine that Stowaway and Jailbird conflict, you don't even get the crime notes Jailbird usually gives you when you start this way. I don't know if the traits I used in-between caused some kind of conflict in the code that caused Jailbird to fail afterwards, but this is what I did on the private copy of the server.
